clarity/tests/arkadiko-freddie-v1-1_test.ts

```diff
@@ -341,36 +342,21 @@ Clarinet.test({
     let result = oracleManager.updatePrice("STX", 77);
     result.expectOk().expectUint(77);
 
-    result = oracleManager.updatePrice("DIKO", 50);
-    result.expectOk().expectUint(50);
-
-    result = vaultManager.createVault(deployer, "DIKO-A", 5, 0.000001)
+    result = vaultManager.createVault(deployer, "STX-A", 5, 0.000001)
     result.expectOk().expectUint(1);
 
     // Should not be able to deposit in STX reserve
     let block = chain.mineBlock([
       Tx.contractCall("arkadiko-freddie-v1-1", "deposit", [
         types.uint(1),
         types.uint(500000000), // 500 STX
-        types.principal("STSTW15D618BSZQB85R058DS46THH86YQQY6XCB7.arkadiko-stx-reserve-v1-1"),
+        types.principal("STSTW15D618BSZQB85R058DS46THH86YQQY6XCB7.arkadiko-sip10-reserve-v1-1"),
         types.principal("STSTW15D618BSZQB85R058DS46THH86YQQY6XCB7.arkadiko-token"),
       ], deployer.address)
     ]);
     block.receipts[0].result
       .expectErr()
       .expectUint(45);
-    
-    // Mint extra
-    result = vaultManager.mint(deployer, 1, 0.000001);
```

Comment:
Q: why did you remove these?

Reply:
In the initial test we created a vault with collateral type DIKO-A, and tried to call deposit/withdraw/mint/burn with wrong parameters. (ex: passing STX reserve to withdraw should fail). But DIKO-A has been removed so test not needed for now.